### DISCUSSION TITLE
Web Search option not displayed in chat since 0.4.3

### DISCUSSION THREAD

**Discussion by vishnualpha** [open-webui/open-webui#7292] [Issues]
## Installation Method

Running in dev mode with npm run dev

## Environment

- v0.4.4

- **Operating System: macOS Big Sur

**Confirmation:**

- [ x] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [x ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

When Web search is enabled in the Admin settings, user should be able to enable web search by clicking + in chat

## Actual Behavior:

Web search option not displayed, only Upload Files available.

## Description

**Bug Summary:**
I have Web search with searxng enabled from admin settings. Till 0.4.2 I was able to see the Web search option in the chat (by clicking + icon) but now the web search option is no longer visible

## Reproduction Details

**Steps to Reproduce:**
1. Enable Web Search from Admin Settings
2. Start a New Chat, and click on the + icon next to the chat input
3. Web search is not displayed

## Logs and Screenshots

**Screenshots/Screen Recordings (if applicable):**
<img width="1176" alt="image" src="https://github.com/user-attachments/assets/7c60a9fb-d32d-488f-9224-593894343821">
<img width="1133" alt="image" src="https://github.com/user-attachments/assets/e1e6dc6b-d489-4abe-a8e9-8a0942431ea7">

**Comment by comeback01** [comment]
By replacing the embedding model with a multilingual one, it works.

I used this model: "embaas/sentence-transformers-multilingual-e5-base". I'm not sure if it's the best choice, but it provides good results.

The default model is designed only for English.
